### DISCUSSION TITLE
Deploy/Kubernetes/Enc

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#166] [General Discussion]
Hi @keshavprasad,
Please check the below error. We are getting this for Deploy/Kubernetes/Enc and Deploy/Kubernetes/Player

TASK [sunbird-deploy : helm upgrade] *******************************************
fatal: [localhost]: FAILED! => {
    "changed": true,
    "cmd": "helm upgrade --install --atomic --timeout 10m enc /var/lib/jenkins/workspace/Deploy/dev/Kubernetes/Enc/kubernetes/helm_charts/core/enc -n dev",
    "delta": "0:10:01.246118",
    "end": "2022-09-15 15:05:17.467417",
    "rc": 1,
    "start": "2022-09-15 14:55:16.221299"
}

STDOUT:

Release "enc" does not exist. Installing it now.

STDERR:

Error: release enc failed, and has been uninstalled due to atomic being set: timed out waiting for the condition

MSG:

non-zero return code

PLAY RECAP *********************************************************************
localhost                  : ok=5    changed=5    unreachable=0    failed=1    skipped=7    rescued=0    ignored=0   

[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // stage
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (slack_notify)
[Pipeline] sh
+ git rev-parse --short HEAD
[Pipeline] sh
+ rev
+ + rev
git name-rev --name-only HEAD
+ cut -d / -f1
[Pipeline] sh
+ echo Deploy/dev/Kubernetes/Enc
[Pipeline] echo
Could not find env specific Slack channel. Check for global slack channel..
[Pipeline] echo
Could not find slack environment variable. Skipping slack notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] ansiColor
[Pipeline] {

[Pipeline] stage
[Pipeline] { (email_notify)
[Pipeline] sh
+ echo Deploy/dev/Kubernetes/Enc
[Pipeline] echo
Could not find env specific email group. Check for global email group..
[Pipeline] echo
Could not find global email group variable. Skipping email notification..
[Pipeline] }
[Pipeline] // stage
[Pipeline] }

[Pipeline] // ansiColor
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 2
Finished: FAILURE

**Comment by beepdot** [comment]
Please refer to the answer posted here - https://github.com/orgs/Sunbird-Ed/discussions/78
